src/cli/node/json_rpc_client.rs

```diff
@@ -99,9 +100,26 @@ pub fn get_balance(addr: SocketAddr, pkh: Option<PublicKeyHash>) -> Result<(), f
     );
     let response = send_request(&mut stream, &request)?;
     log::info!("{}", response);
-    let amount = parse_response::<u64>(&response)?;
-
-    println!("{} wits", Wit::from_nanowits(amount));
+    let amount = parse_response::<NodeBalance>(&response)?;
+    if amount.total >= amount.confirmed {
+        println!(
+            "Total balance:       {} wits\n\
+        Confirmed balance:   {} wits\n\
+        Pending balance:     {} wits",
+            Wit::from_nanowits(amount.total),
+            Wit::from_nanowits(amount.confirmed),
+            Wit::from_nanowits(amount.total - amount.confirmed)
+        );
+    } else {
+        println!(
+            "Total balance:       {} wits\n\
+        Confirmed balance:   {} wits\n\
+        Pending balance:     -{} wits",
+            Wit::from_nanowits(amount.total),
+            Wit::from_nanowits(amount.confirmed),
+            Wit::from_nanowits(amount.confirmed - amount.total)
+        );
+    }
```

Comment:
There's some code repetition here, I think we could unify both branches, as `total - confirmed` will be negative for all cases in which `total < confirmed`. Although maybe you are doing this to avoid casting them to `i64`, right? If so, I guess ths is fine then.

On other matters, I'm still unsure about the "Total balance" thing. Beyond my initial concern that the wording "total" doesn't represent what it actually is (what your balance will be when pending transactions are eventually confirmed), I have the feeling that this still could lead someone into thinking they have received a payment while it is still not confirmed. Maybe I'm nitpicking here, but I think we're assuming the user to be educated enough about finality of transactions, when the truth is that the demographics for Witnet node operators are quite diverse and many of them won't totally understand what it means.

My hunch is that showing confirmed and pending balance may be enough. If someone has a use case in which they  do need to know their potential balance after eventual confirmation, I think they can do the math on their own.

However, if we finally decide to keep the total balance, in all cases, I would totally print it last, as in that way it'll be easier for the user to figure out that it's in fact the addition of the confirmed and pending balances.

Reply:
I think is alright not to include the Total balance, but in that case I would specify the sign of the pending balance (as it is now),to avoid confusion

Reply:
Good. I was assuming the sign would appear automatically when printing a negative `i64` number.

Reply:
I've added a function (`negative_wits`) to print a negative i64 in the case the pending balance is < 0. Another option could be change the Wit structure from u64 to i64. However, since at the moment it looks like it only affects this specific case, i think adding the function `negative_wits` should be enough